src/Actions/OutputsDslQueryAction.php

```diff
@@ -96,4 +101,26 @@ private function mapCondition(array $condition): array
 
         return $parser->parse($field, $value);
     }
+
+    private function getMappings(): array
+    {
+        $indexName = config('rapidez.es_prefix').'_products_'.config('rapidez.store');
+        $esMappings = ElasticSearch::indices()->getMapping(['index' => $indexName]);
+        $mappings = data_get($esMappings, '*.mappings.properties', []);
+
+        return count($mappings) ? array_pop($mappings) : [];
```

Comment:
Not sure why this is needed, but:

```suggestion
        return Arr::last($mappings) ?? [];
```

This should do the same thing and seems cleaner to me.

Reply:
Agreed!